ntoskrnl/kdbg/kdb.c

```diff
@@ -1605,11 +1605,38 @@ KdbEnterDebuggerException(
             ExceptionRecord && ExceptionRecord->NumberParameters != 0)
         {
             /* FIXME: Add noexec memory stuff */
+<<<<<<< HEAD
```

Comment:
Hum!

Reply:
Yes, I noticed it too late, I'm fixing...